tox.ini

```diff
@@ -51,9 +51,11 @@ commands =
   coverage combine
   coveralls
 
-[testenv:pep8]
-deps = pep8
-commands = pep8 --exclude=luigi/six.py --ignore E501,E402,E731 luigi test examples bin
+[testenv:flake8]
+deps = flake8
+commands = flake8 --exclude=doc,test,luigi/six.py --ignore=E501
+  flake8 --ignore=E265,E501 doc
+  flake8 --ignore=E501,F401,F841 test
```

Comment:
Of course. What line length should be enough?

There are lines > 200 symbols length.

Here are some stats.

```
(luigi)luigi 🖖 flake8 --select=E501 --max-line-length=100 | wc -l
     512
(luigi)luigi 🖖 flake8 --select=E501 --max-line-length=120 | wc -l
     166
(luigi)luigi 🖖 flake8 --select=E501 --max-line-length=160 | wc -l
      28
(luigi)luigi 🖖 flake8 --select=E501 --max-line-length=200 | wc -l
       9
```

F401,F841 should also be fixed. Can we do it in separate pull request as well as line length?